hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/ContainerTestUtils.java

```diff
@@ -113,17 +113,23 @@ public static EndpointStateMachine createEndpoint(Configuration conf,
         new LegacyHadoopConfigurationSource(conf));
   }
 
-  public static OzoneContainer getOzoneContainer(
+  public static OzoneContainer getOzoneContainerWithMockContext(
```

Comment:
I suggest keeping the original name?

* reduce change
* there is no `getOzoneContainer` (without mock context) method we need to differentiate from

Reply:
Done.